hibernate-rx-core/src/test/java/org/hibernate/rx/containers/PostgreSQLDatabase.java

```diff
@@ -5,6 +5,10 @@
 public class PostgreSQLDatabase {
 
 	private static final boolean USE_DOCKER = Boolean.getBoolean( "docker" );
+	
+	public static final String USERNAME = "hibernate-rx";
+	public static final String PASSWORD = "hibernate-rx";
+	public static final String DB_NAME = "hibernate-rx";
```

Comment:
I would like these credentials to be the same for all the databases, maybe we can move them in an interface. I wasn't planning to do it in this PR but you are already moving them to constants, so ...

Reply:
This might be possible for the current DBs, but for DB2 this will not work because DB_NAME cannot contain dash chars, and the username cannot contain dash chars and cannot exceed length 8.
I can move them to an interface now, but we will need to adjust the values as we support more DBs in the future with different requirements (such as DB2)

Reply:
Let's wait then and treat it as a separate issue